### PR TITLE
YES_42 Star ratings are missing on product pages Resolved.

### PR DESCRIPTION
**Pull request by ijazalise**
resolved

### CODE REVIEW

File: app/code/community/Yotpo/Yotpo/Helper/Data.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
         }
         if ($block != null) {
             if ($print == true) {
-                $block->toHtml();
```
Comment (ohadsc) [inline app/code/community/Yotpo/Yotpo/Helper/Data.php]: can remove if and logic be the same

File: app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+class Yotpo_Yotpo_Block_Catalog_Product_View extends Mage_Catalog_Block_Product_View {
+
+    const CATEGORY_BOTTOMLINE = 'yotpo/yotpo_general_group/enable_bottomline_category_page';
+
+    public function getReviewsSummaryHtml(Mage_Catalog_Model_Product $product, $templateType = false, $displayIfNoReviews = false) {
+
+        $enableBottomlineCategoryPage = Mage::getStoreConfig(self::CATEGORY_BOTTOMLINE);
```
Comment (ohadsc) [inline app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php]: The config here is seems to be the same as in category page (list.php), is it on purpose?  Isn't this view rendering on the product page?

File: app/code/community/Yotpo/Yotpo/Helper/Data.php (hunk 1 above)
Comment (ijazalise) [reply app/code/community/Yotpo/Yotpo/Helper/Data.php]: yes we can remove this.

File: app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php (hunk 2 above)
Comment (ijazalise) [reply app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php]: enable/disable configuration are same throughout the module. Here we are extending Block of Product for Reviews While at category page we are extending product listing.

File: app/code/community/Yotpo/Yotpo/Helper/Data.php (hunk 1 above)
Comment (ohadsc) [reply app/code/community/Yotpo/Yotpo/Helper/Data.php]: @ijazalise could you please remove this redundant code?

**Comment by ijazalise** [comment]
removing this code.

File: app/code/community/Yotpo/Yotpo/Helper/Data.php [hunk 3]
```diff
@@ -53,14 +53,7 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
 
         if ($product != null) {
             $block->setAttribute('product', $product);
-        }
```
Comment (ohadsc) [inline app/code/community/Yotpo/Yotpo/Helper/Data.php]: @ijazalise my mistake, please return the logic as it was. sorry for confusing you

Comment (ijazalise) [reply app/code/community/Yotpo/Yotpo/Helper/Data.php]: ok

File: app/code/community/Yotpo/Yotpo/Helper/Data.php [hunk 4]
```diff
@@ -54,9 +54,9 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
         if ($product != null) {
             $block->setAttribute('product', $product);
         }
-        if ($block != null) {
+		if ($block != null) {
             if ($print == true) {
-                $block->toHtml();
+               return $block->toHtml();
```
Comment (ohadsc) [inline app/code/community/Yotpo/Yotpo/Helper/Data.php]: @ijazalise can we just leave it as it was? i see here additional return that you've added. unless I am missing something and this code is relevant to your changes

File: app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php (hunk 2 above)
Comment (ohadsc) [reply app/code/community/Yotpo/Yotpo/Block/Catalog/Product/View.php]: 👍

File: app/code/community/Yotpo/Yotpo/Helper/Data.php (hunk 4 above)
Comment (ohadsc) [reply app/code/community/Yotpo/Yotpo/Helper/Data.php]: @ijazalise ?

**Comment by ijazalise** [comment]
Hi Ohadsc,
Hope you are doing well.
I have done the changes you requested before. Now what changes you want kindly clear me.
Regards,
Ijaz Ali

**Comment by ohadsc** [comment]
@ijazalise please remove the 'return' from 'return $block->toHtml();' 
let me know if it's not understood, my intention is that it will be the same as it was before the changes on the pull request

**Comment by ijazalise** [comment]
Hi ohadsc,
I had implemented.

**Comment by ohadsc** [comment]
@ijazalise thanks, from some reason the pull request was closed. I reopened it and approved.
Thanks again, and sorry for all the miss communications

**Comment by billykov** [comment]
@ohadsc could we please merge, QA and release this - it was escalated by Support
https://jira.yotpo.com/browse/YO-9083

**Comment by danadavid** [comment]
👍

**Pull request merged**